### PR TITLE
go-config -> viper

### PR DESCRIPTION
**Pull request by rigelrozanski**
Replace go-config with viper, In conjunction with tendermint PR https://github.com/tendermint/tendermint/pull/442

### CODE REVIEW

**Review by melekes — APPROVED** [review]

File: switch_test.go
Comment (ebuchman) [inline switch_test.go]: lets keep the `cmn`.

**Review by ebuchman — COMMENTED** [review]
Cool. Should probably wait until the more-tests stuff is merged as that changes a lot re config

File: switch_test.go
Comment (rigelrozanski) [inline switch_test.go]: Ok yeah, I just deleted because there were go-common had been imported twice from the last commit... and all other go-common was using the `.` imported, I'll change now though

**Pull request closed without merging**
